File: Baekjoon/Class1/26.2562 최댓값.cpp
#include <iostream>
using namespace std;

int main(void) {
	ios_base::sync_with_stdio(false);
	cin.tie(NULL);

	int input, max = -1, max_index = -1;
	for (int i = 1; i <= 9; i++) {
		cin >> input;
		if (input > max) {
			max = input;
			max_index = i;
		}
	}
	cout << max << '\n';
	cout << max_index << '\n';
	return 0;
}

File: Baekjoon/Class1/26.2562 최댓값.java
import java.io.*;

class Main{
    public static void main(String[] args) throws IOException {
        int input, max = -1, maxIndex = -1;
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();
        for(int i = 1; i <= 9; i++) {
            input = Integer.parseInt(br.readLine());
            if(input > max) {
                max = input;
                maxIndex = i;
            }
        }
        sb.append(max).append('\n');
        sb.append(maxIndex).append('\n');
        System.out.print(sb);
        br.close();
    }
}